examples/tutorials/asr_inference_with_ctc_decoder_tutorial.py

```diff
@@ -222,28 +224,212 @@ def forward(self, emission: torch.Tensor) -> str:
 #
 # Now that we have the data, acoustic model, and decoder, we can perform
 # inference. Recall the transcript corresponding to the waveform is
-# ``"when it was the seven hundred and eighteenth night"``
+# ``"i really was very much afraid of showing him how much shocked i was at some parts of what he said"``
 #
 
+actual_transcript = "i really was very much afraid of showing him how much shocked i was at some parts of what he said"
 emission, _ = acoustic_model(waveform)
 
+
+######################################################################
+# The greedy decoder give the following result.
+#
+
+greedy_result = greedy_decoder(emission[0])
+greedy_transcript = greedy_result.replace("|", " ").lower().strip()
```

Comment:
I think `.replace("|", " ").lower().strip()` and `.split()` applied later can be put inside of `GreedyDecoder.forward` method.